# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ipmiutil/no-gzip-manpages.patch
# Copyright (C) 2007 - 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Do not gzip all available man pages

--- ipmiutil-2.6.8/doc/Makefile.orig	2010-08-16 14:40:23.760171932 +0200
+++ ipmiutil-2.6.8/doc/Makefile	2010-08-16 14:40:34.076006988 +0200
@@ -67,5 +67,4 @@
 	${INSTALL_SCRIPT} ipmi_port.sh ${initto}/ipmi_port
 	for f in ${sbinfls}; do \
 	    ${INSTALL_SCRIPT} $$f ${sbinto}; done
-	cd ${manto}; gzip -f *.8
 	
